### ISSUE TITLE
Referring to Backpack as Host

### ISSUE THREAD

**Issue by kayaelle** [WebOfTrustInfo/rebooting-the-web-of-trust-spring2018#68]
In reference to: https://github.com/WebOfTrustInfo/rebooting-the-web-of-trust-spring2018/blob/master/draft-documents/open-badges-are-verifiable-credentials.md

In various places, the backpack is referred to as a host. Calling the backpack a host could mistakenly cause issuers to think that if the badges they issue are displayed in the earner's backpack accounts that they no longer need to host the json files or keys. 

The backpack should be referred to as a displayer or an example of a displayer. Implying that it has more functionality than that in terms of verification would be wrong especially in this context.

**Comment by ottonomy** [comment]
I agree that this terminology may be confusing, especially for "host". Unfortunately, some folks objected to the term "backpack" last summer (connotations of childlike environments I guess), so when it came time to set up the conformance requirements, the three OB roles certified are "issuer", "host" and "displayer". I agree that host has the specific issue you mention, so don't know how to resolve. All existing certified hosts are also certified displayers, so probably the existing divisions aren't too useful. However, some issuers are also displayers without being hosts. Host is really *only* about being able to import from elsewhere (and re-export as needed) and doesn't imply any display functionality. Can't find the link to the Google Doc draft of conformance unfortunately. @jbohrer is this published in html/officially yet?

**Comment by kayaelle** [comment]
How about using "consumer" or "consumption app"?  I think this broadly applies to any app or system that reads the data and uses it whether it be displaying or processing it for other uses. I would suggest that the three OB roles are: issuer, earner, and consumer.
